src/utils/pgwire/src/pg_extended.rs

```diff
@@ -406,7 +406,7 @@ impl PreparedStatement {
                         .to_string(),
                     Format::Text => cstr_to_str(raw_param).unwrap().to_string(),
                 },
-                DataType::Int64 => {
+                DataType::Int64 | DataType::Serial => {
```

Comment:
Better use an error. We do not expect an input parameter of type `serial` as part of prepared statement.